ui/vty-lib/src/Ariadne/UI/Vty/Widget.hs

```diff
@@ -173,6 +173,7 @@ data WidgetEvent
   | WidgetEventEditLocationChanged
   | WidgetEventListSelected Int
   | WidgetEventCheckboxToggled
+  | WidgetEventNoModal
```

Comment:
I would rather name it `ModalExited`, because the semantics of this ADT is "something happened"